cmd/gluster-virtblock/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gvb "github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock"
+	"github.com/gluster/gluster-csi-driver/pkg/gluster-virtblock/config"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	// #nosec
+	_ = flag.Set("logtostderr", "true")
+}
+
+func main() {
+	// #nosec
+	_ = flag.CommandLine.Parse([]string{})
+	var config = config.NewConfig()
+
+	cmd := &cobra.Command{
+		Use:   "gluster-virtual-block-csi-driver",
+		Short: "GlusterFS Virtual Block CSI driver",
```

Comment:
There is no point in keeping `GlusterFS` string here, `Gluster` fit better

Reply:
Done